app/layout.tsx

```diff
@@ -5,8 +5,8 @@ import { Inter } from "next/font/google";
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: "게임왜건",
+  description: "뭘적어야하지",
```

Comment:
```suggestion
  description: "게임 플랫폼 정보 사이트",
```
해당 부분 위 사이트에 대한 정보를 적는건 어떤가용 ?
저는 이런 내용으로 추천 드립니다~

Reply:
확인했습니다!